portiaui/app/components/list-item-link-crawling.js

```diff
@@ -33,7 +33,11 @@ export default Ember.Component.extend({
 
     actions: {
         saveSpider() {
-            this.get('spider').save();
+            this.get('spider').save().then(() => {
+                if (this.get('linksToFollow') !== 'none') {
```

Comment:
When they set it to all they can set the nofollow rule too. Should I change to to just open for patterns?